src/model/question.ts

```diff
@@ -54,6 +54,20 @@ export abstract class QuestionTemplate {
     }
 }
 
+// TODO: hacky? used when we already have a multiple choice question generated but
+// we still want to make a QuestionEvent
```

Comment:
This is fine, or you could change the event to have Template | Question instead of just Template